lms/djangoapps/courseware/plugins.py

```diff
@@ -0,0 +1,153 @@
+"""Course app config for courseware apps."""
+from typing import Dict, Optional
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.utils.translation import ugettext_noop as _
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.course_apps.plugin import CourseApp
+from openedx.core.lib.courses import get_course_by_id
+
+User = get_user_model()
+
+TEXTBOOK_ENABLED = settings.FEATURES.get("ENABLE_TEXTBOOK", False)
```

Comment:
nit: possible use-case for using `SettingDictToggle ` 
https://github.com/edx/edx-toggles/blob/master/edx_toggles/toggles/internal/setting_toggle.py#L24

Reply:
If I add that I will also need to add toggle documentation (that should only be in one place), and I don't know what to add there. If you could help me with that I'll move this to a toggles.py file as well.

Reply:
Sure, Let me take a look and see how I can help. Thanks for bringing this up!